tests/simulate/test_simulate.py

```diff
@@ -0,0 +1,120 @@
+"""Unit tests for simulate module."""
+import unittest
+
+import numpy as np
+
+import relentless
+
+class test_Dilute(unittest.TestCase):
+    """Unit tests for relentless.Dilute"""
+
+    #mock functions for use as operations
+    def update_ens(self, ensemble, potentials, options):
+        for o in options:
+            if o=='constant_ens' and options[o]:
+                return
```

Comment:
Thanks to the trick you taught me, this could be done more compactly:
```suggestion
        if options.get('constant_ens',False):
            return
```